examples/constrained/multiclass_linear_svm.py

```diff
@@ -102,8 +102,8 @@ def matvec_G(_, beta):
                       params_eq=(None, b),
                       params_ineq=(None, h))
   
-  init_params = osqp._box_osqp.init_params(init_x=None, **hyper_params)
-  sol, _ = osqp.run(init_params=None, **hyper_params)
+  init_params = osqp.init_params(init_x=None, **hyper_params)
+  sol, _ = osqp.run(init_params=init_params, **hyper_params)
```

Comment:
Since you take care of the `init_params is None` case in `osqp.py`, can't you just do 
```python
sol, _ = osqp.run(**hyper_params)
```

Reply:
The issue with the `init_params()` method is that it's not really part of our standard API so it will be surprising for some users.